clustered/clustered-dist/src/assemble/server/conf/tc-config.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<tc-config xmlns="http://www.terracotta.org/config"
+           xmlns:ohr="http://www.terracotta.org/config/offheap-resource">
+  <!--
+    This is the default Terracotta server configuration file for the Ehcache kit.
+
+    It defines a single offheap resource of 512MB to be used for caching data.
+
+    It also defines a single server, but you can add another one to benefit from high availability.
+  -->
+
+  <plugins>
+    <config>
+      <ohr:offheap-resources>
+        <ohr:resource name="main" unit="MB">512</ohr:resource>
+        <!-- You can define extra offheap resource if you want isolated resource -->
+        <!--<ohr:resource name="other" unit="MB">128</ohr:resource>-->
+      </ohr:offheap-resources>
+    </config>
+  </plugins>
+
+  <servers>
+    <server host="localhost" name="default-server">
+      <!--
+        Note that relative path will be resolved from the location of this configuration file.
+        Also if you end up running more than one server on the same machine for testing purpose, make sure these are different.
+       -->
+      <logs>logs</logs>
+      <!--
+        This port is used by clients to communicate to the server.
+        Its value is actually the default one and could be omitted.
+      -->
+      <tsa-port>9410</tsa-port>
+      <!--
+        This port is used for server to server communication.
+        Its value is actually the default one and is thus omitted.
+      -->
+      <!--<tsa-group-port>9430</tsa-group-port>-->
```

Comment:
What?  Why do we explicitly configure the tsa port to the same value as it's default.  But we let this we be the default.  In other words why do these two sentences conflict:

* Its value is actually the default one and could be omitted.
* Its value is actually the default one and is thus omitted.